### ISSUE TITLE
Option to roll elastigroup on update

### ISSUE THREAD

**Issue by kmcgrath** [SungardAS/spotinst-lambda#3]
When an elastigroup is updated have an option to also roll the group immediately.  By default when an elastigroup is updated Spotinst will not change running instances until one needs to be replaced.

**Comment by kmcgrath** [comment]
The CloudFormation definition will look like the following where `group` and `group_roll` will be defined as properties.  When `UPDATE` is called and `update_roll` is defined the lambda function will execute a roll on a successful `group` update.

```
{
  "accessToken": TOKEN
  "requestType": "create",
  "resourceType": "elasticgroup",
  "group": { },
  "group_roll": {}
}
```

**Comment by kmcgrath** [comment]
Implemented with PR #7
